### ISSUE TITLE
More or better links to election pages

### ISSUE THREAD

**Issue by JoeMitchell** [DemocracyClub/WhoCanIVoteFor#134]
Jag expected there to be a link here:

![image](https://cloud.githubusercontent.com/assets/8096305/26277372/cd3fc80e-3d7d-11e7-88de-abd2d0068164.png)

Because he didn't want to share his postcode.

Presumably if there was a link, it'd go to the constituency page via /election, but there's some funny stuff going on there...

![image](https://cloud.githubusercontent.com/assets/8096305/26277380/e4a780ea-3d7d-11e7-827e-1b123db102b2.png)

**Comment by pmk01** [comment]
This is still valid. 
Also:
![Screenshot 2020-12-22 110013](https://user-images.githubusercontent.com/61510609/102881563-f42d3380-4444-11eb-97c1-b4db0921ba15.png)

**Comment by VirginiaDooley** [comment]
> This is still valid. Also: ![Screenshot 2020-12-22 110013](https://user-images.githubusercontent.com/61510609/102881563-f42d3380-4444-11eb-97c1-b4db0921ba15.png)

I think this is ok? https://whocanivotefor.co.uk/elections/TE1%201ST/

**Comment by pmk01** [comment]
Thanks! It is better than ok. 

It's not an answer to the original issue, though, which is devising an easy way to share the ballot without sharing your postcode. Reading back over the tweets, I like the idea of a 'share this ballot' button. However, I think that's a bigger piece of work. Perhaps the answer is to write that up and close this issue?

**Comment by pmk01** [comment]
Yes, you got it. As things stand, there's no obvious way to find a link to the ballot. The original issue suggested simply linking the ballot title text, but on reflection I'm not sure that's the best way of telling people they can share the ballot. A button would be clearer, I think. This is probably something we need to talk over.

**Comment by pmk01** [comment]
Oh right, sorry. I want people to be able to share the ballot. I hadn't thought about the multi-ballot option. That sounds complicated...

**Comment by GeoWill** [comment]
It is. Sharing the ballot is fine, and what Virginia is doing in #1682 should address that. Admittedly it will require c&p a link for now, but adding a button can happen later/needs design thought as you say. The multi ballot option is best served by a postcode imo.
